api/v1/mapping/src/commonMain/kotlin/Mappings.kt

```diff
@@ -761,3 +766,13 @@ fun ApiSourceCodeOrigin.mapToModel() =
 fun User.mapToApi() = ApiUser(username, firstName, lastName, email)
 
 fun EcosystemStats.mapToApi() = ApiEcosystemStats(name = name, count = count)
+
+fun VulnerabilityRating.mapToApi() = ApiVulnerabilityRating.valueOf(name)
+
+fun VulnerabilityWithAccumulatedData.mapToApi() = ApiVulnerabilityWithAccumulatedData(
```

Comment:
We already have `EcosystemStats`, so maybe call this `VulnerabilityStats`?

Reply:
The only thing is, I wonder if `VulnerabilityStats` would be a bit confusing as `EcosystemStats` combines info from multiple packages, and here this provides more info about a single vulnerability (or actually a Vulnerability+Identifier pair). So maybe `VulnerabilityStats` would be better for combining info for multiple vulnerabilities. But I agree that the `VulnerabilityWithAccumulatedData` is kind of a bad name :smiley: Maybe something like `VulnerabilitySummary`?

Reply:
"accumulated data" is very vague, but "stats" actually is as well. So how about something like `ProductVulnerability` or `ProductVulnerabilitySummary`, then the upcoming class for organizations could be named similarly.